### ISSUE TITLE
Some heatMap function signatures are broken

### ISSUE THREAD

**Issue by longouyang** [probmods/webppl-viz#45]

**Comment by longouyang** [comment]
These work:

``` js
// samples of objects
viz.heatMap(repeat(2e2, function() { return {a: gaussian(0,1), b: beta(0.5,0.5)} }) )
// dist: supply n as option
viz.heatMap(MH(function() { return {a: gaussian(0,1), b: beta(0.5,0.5)} }, 2e2),
            {n: 2e2})
```

These don't:

``` js
// non-object-y format stopped working
viz.heatMap(repeat(2e2, function() { [gaussian(0,1), beta(0.5,0.5)] }) )

// n is not actually optional for dist signature
viz.heatMap(MH(function() { return {a: gaussian(0,1), b: beta(0.5,0.5)} }, 2e2))
```

**Comment by longouyang** [comment]
Fixed by 5e85c4a, a24cbf4
